src/Estimators/src/BaseEstimatorFromFootIMU.cpp

```diff
@@ -11,12 +11,12 @@
 
 using namespace BipedalLocomotion;
 
-Eigen::Vector3d toZYX(const Eigen::Matrix3d& r)
+Eigen::Vector3d toXYZ(const Eigen::Matrix3d& r)
 {
     Eigen::Vector3d output;
-    double& thetaZ = output[0]; // Roll
+    double& thetaX = output[0]; // Roll
     double& thetaY = output[1]; // Pitch
-    double& thetaX = output[2]; // Yaw
+    double& thetaZ = output[2]; // Yaw
```

Comment:
How comes that you changed the order of the angles and the computations remain the same? Was it bugged before?